includes/class-sensei-utils.php

```diff
@@ -1945,7 +1948,22 @@ public static function update_course_status( $user_id, $course_id, $status = 'in
 					update_comment_meta( $comment_id, $key, $value );
 				}
 			}
-			do_action( 'sensei_course_status_updated', $status, $user_id, $course_id, $comment_id );
+
+			/**
+			 * Fires when a course status is updated.
+			 *
+			 * @hook sensei_course_status_updated
+			 *
+			 * @since 1.7.0
+			 * @since $$next-version$$ $old_status parameter added.
+			 *
+			 * @param string      $status          The status.
+			 * @param int         $user_id         The user ID.
+			 * @param int         $course_id       The course ID.
+			 * @param int         $comment_id      The comment ID.
+			 * @param string|null $previous_status The old status. Null if previous status was not set.
```

Comment:
Should we also change this for consistency?

```suggestion
			 * @param string|null $previous_status The previous status. Null if previous status was not set.
```